### DISCUSSION TITLE
Changing the colour of a single Column if certain Criteria is met

### DISCUSSION THREAD

**Discussion by Dennis-Scheurer** [Live-Charts/LiveCharts2#969] [Q&A]
Hello everyone,

I was wondering if it is possible to change the colour of a single Column within a Columnseries.

For example: If the value ist greater than 5, the colour should be red.

I tried my best, but couldn't find a way to access a single Column within the series.

**Comment by Brinvest** [comment]
1+ Also looking for this, using WPF. I did manage to sort out a partial solution using a different ColumnSeries with a single data point for each, but this method seems to totally break the tooltip functionality. The entire graph responds as one when you mouse over, and tooltip info for every series tries to display in a single tooltip! Clearly not the correct approach.

Lots of google results with solutions using CartesianMappers for the original Livecharts.Wpf, but have not been able to get it working for the nuget package LiveChartsCore.SkiaSharpView.WPF (2.0.0-beta.701)

Please Help!
